stripe-patrons-data/src/main/scala/com/gu/patrons/services/ConfigService.scala

```diff
@@ -7,9 +7,11 @@ import com.typesafe.scalalogging.StrictLogging
 trait ConfigService extends StrictLogging {
 
   protected def findParameterOrThrow(name: String, params: List[Parameter]) =
-    findParameterValue(name, params).getOrElse(
-      throw new RuntimeException(s"Missing config value for parameter $name"),
-    )
+    findParameterValue(name, params).getOrElse {
+      val error = s"Missing config value for parameter $name"
+      logger.error(error)
```

Comment:
this log doesn't show up in cloudwatch if it fails during the initialisation stage

Reply:
good point - where do the logs/stderr go from the initial startup/bake at deploy time?

Reply:
sorry I should have replied to my original comment!
I reported this issue to AWS and they've since fixed it.
So errors during initialisation _do_ now show in cloudwatch logs as normal.
So if a riffraff deploy fails, you can find the relevant logs fairly easily